src/components/navbar/index.tsx

```diff
@@ -30,12 +57,20 @@ const Navbar: React.FC = (): JSX.Element => {
       <NavLogo
         to="/"
         onClick={(): void => {
-          setTutorialDisplay(false);
-          setSelectorDisplay(true);
+          setTutorialDisplay(true);
+          setSelectorDisplay(false);
```

Comment:
can you move this into a defined function? React has a performance penalty for functions defined inline